index.js

```diff
@@ -348,6 +348,20 @@ exports.hashPersonalMessage = function (message) {
 }
 
 /**
+ * Returns the address corresponding to the `personal_sign`-formatted `signature` for the `message`.
+ * @param message
+ * @param signature
+ * @returns {Buffer} hash
+ */
+
+exports.recoverPersonalMessage = function (message, signature) {
```

Comment:
Since we require the original message here, wouldn't it make sense supplying an address and making this a `verifyPersonalMessage`?

`recoverPersonalMessage` wounds a bit like a message being recovered, while we're recovering the signing address.